app/views/appliances/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Appliances
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= Appliance.count %> appliances!
```

Comment:
Take a look at this other [comment](https://github.com/Lohenrex/mogu-mogu/pull/13#discussion_r794122375) to fix this :)

Reply:
Already fixed in the latest commit!